### PR TITLE
doc improvement: Update samples and apps with Thingy:91 X

### PR DESCRIPTION
**Pull request by divipillai**
* Update samples and apps with Thingy:91 X, for the samples and apps the support are added.
* Asset Tracker v2
* Connectivity bridge
* Serial LTE modem
* Memfault sample
* MQTT sample

### CODE REVIEW

File: doc/nrf/links.txt [hunk 1]
```diff
@@ -1376,6 +1376,7 @@
 
 .. ### Source: bosch-sensortec.com
 
+.. _	: https://www.bosch-sensortec.com/products/environmental-sensors/gas-sensors/bme688/
```
Comment (MarkusLassila) [inline doc/nrf/links.txt:1379]: Something missing?

File: applications/serial_lte_modem/doc/slm_testing.rst [hunk 2]
```diff
@@ -1503,10 +1503,10 @@ Complete the following steps to test the functionality provided by the :ref:`SLM
 TWI AT commands
 ***************
 
-Complete the following steps to test the functionality provided by the i2c sensors on the Thingy:91 using the two-wire interface (TWI):
+Complete the following steps to test the functionality provided by the i2c sensors on the Thingy:91 and Thingy:91 X using the two-wire interface (TWI):
```
Comment (wiba-nordic) [inline applications/serial_lte_modem/doc/slm_testing.rst:1506]:
```suggestion
Complete the following steps to test the functionality provided by the i2c sensors on the Thingy:91 or Thingy:91 X using the two-wire interface (TWI):
```

File: applications/serial_lte_modem/doc/slm_testing.rst [hunk 3]
```diff
@@ -1503,10 +1503,10 @@ Complete the following steps to test the functionality provided by the :ref:`SLM
...
 
 1. Test the TWI list command using ``AT#XTWILS``.
-   As Thingy:91 connects to the sensors via i2c2, it shows that TWI2 is available:
+   As the devices connect to the sensors through i2c2, it shows that TWI2 is available:
```
Comment (wiba-nordic) [inline applications/serial_lte_modem/doc/slm_testing.rst:1509]:
```suggestion
   As the device connects to the sensors through i2c2, it shows that TWI2 is available:
```

File: samples/debug/memfault/README.rst [hunk 4]
```diff
@@ -8,7 +8,7 @@ Memfault
    :depth: 2
 
 The Memfault sample shows how to use the `Memfault SDK`_ in an |NCS| application to collect coredumps and metrics.
-The sample connects to an LTE network using an nRF91 Series DK or a Thingy:91, or to Wi-Fi®  using the nRF7002 DK, and sends the collected data to Memfault's cloud using HTTPS.
+The sample connects to an LTE network using an nRF91 Series DK or a Thingy:91, or a Thingy:91 X, or to Wi-Fi®  using the nRF7002 DK, and sends the collected data to Memfault's cloud using HTTPS.
```
Comment (wiba-nordic) [inline samples/debug/memfault/README.rst:11]:
```suggestion
The sample connects to an LTE network using an nRF91 Series DK, a Thingy:91, or a Thingy:91 X, or to Wi-Fi®  using the nRF7002 DK, and sends the collected data to Memfault's cloud using HTTPS.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]

**Review by MarkusLassila — APPROVED** [review]
SLM ok

**Review by wiba-nordic — APPROVED** [review]

**Review by maxd-nordic — APPROVED** [review]

**Pull request merged**